4-storage/relational-database-shredder/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+log4j.rootCategory=WARN, file
```

Comment:
Is this file required by Spark for testing? Can we add a comment line to it explaining the rationale for adding?